.github/workflows/pr-container-build.yaml

```
@@ -22,7 +22,10 @@ on:
   # will prevent the job from running until it's approved manually by human intervention.
   pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
-    branches: [ main ]
+    branches: 
+    - main
+    - rhdh-1.**
+    - 1.**.x
```

For the same reasons as above:

```suggestion
    - 1.[0-9]+.x
```